Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json

```diff
@@ -0,0 +1,630 @@
+                                        "name": "result_output_ip",
+                                        "type": "array"
+                                    }
+                                ]
+                            }
+                        },
+                        "Initializa_IP_Comment": {
+                            "runAfter": {
+                                "Init_Result_IP": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "IPcomment",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        },
+                        "MDTI-Base": {
+                            "runAfter": {},
+                            "type": "Workflow",
+                            "inputs": {
+                                "host": {
+                                    "triggerName": "manual",
+                                    "workflow": {
+                                        "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Logic/workflows/MDTI-Base')]"
+                                    }
+                                }
+                            }
+                        },
+                        "intialize_domain_comment": {
+                            "runAfter": {
+                                "Init_Result_Host": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Domain_comment",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "azuresentinel": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('AzureSentinelConnectionName'))]",
+                                "connectionName": "[variables('AzureSentinelConnectionName')]",
+                                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]"
+                            }
```

Comment:
Azure sentinel connection  should be a "ManagedServiceIdentity" .

for reference kindly refer to :
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/azuredeploy.json